runtime/compiler/vimdoc.vim

```diff
@@ -0,0 +1,19 @@
+" Vim Compiler File
+" Language:             vimdoc
+" Maintainer:           Wu, Zhenyu <wuzhenyu@ustc.edu>
+" Latest Revision:      2024-04-09
+"
+" you can get it by `pip install vimdoc` or the package manager of your distribution.
+
+if exists('b:current_compiler')
+  finish
+endif
+let b:current_compiler = 'vimdoc'
+
+let s:save_cpoptions = &cpoptions
+set cpoptions&vim
+
+CompilerSet makeprg=vimdoc
```

Comment:
this `vimdoc` mostly would not be easy getting from most distr, and it req a plugin name as its arg, this is a problem.

Reply:
Yes, I also think upstream should add a default path as its arg.